flake.nix

```diff
@@ -4,7 +4,7 @@
 
   # specify all input dependencies needed to create the outputs of the flake
   inputs = {
-    nixpkgs.url = "github:nixos/nixpkgs?ref=nixos-unstable";
+    nixpkgs.url = "github:nixos/nixpkgs?ref=24.05";
```

Comment:
Sneaking this in, I'd rather be pinned to some stable packages than use unstable, unless there was a reason to do so?